pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/JobConfigConstants.java

```diff
@@ -59,4 +59,5 @@
   // This setting should be used if you will generate less # of segments after
   // push. In preprocessing, this is likely because we resize segments.
   public static final String DELETE_EXTRA_SEGMENTS = "delete.extra.segments";
+  public static final String BOOTSTRAP_JOB = "job.bootstrap";
```

Comment:
Recommend using something more specific such as "use.relative.path". I don't see a clear connection between bootstrap and using relative path.

Reply:
done